experiments/pl_baal_example.py

```diff
@@ -0,0 +1,178 @@
+import sys
+import copy
+from abc import ABC, abstractmethod
+from collections import OrderedDict
+
+from collections.abc import Sequence
+
+from typing import Dict, Any
+
+import numpy as np
+import structlog
+import torch
+from pydantic import BaseModel
+from pytorch_lightning import LightningModule, Trainer, Callback
+from torch import optim
+from torch.nn import CrossEntropyLoss
+from torch.utils.data import DataLoader
+from torchvision.datasets import CIFAR10
+from torchvision.models import vgg16
+from torchvision.transforms import transforms
+from tqdm import tqdm
+
+from baal.active import ActiveLearningDataset, ActiveLearningLoop
+from baal.active.heuristics import BALD
+from baal.modelwrapper import mc_inference
+from baal.utils.cuda_utils import to_cuda
+from baal.utils.iterutils import map_on_tensor
+
+from baal.utils.pytorch_lightning import ActiveLearningMixin, ResetCallback, BaalTrainer
+
+log = structlog.get_logger('PL testing')
+
+
+class VGG16(LightningModule, ActiveLearningMixin):
```

Comment:
Need to flip the inheritance I think? So that ActivelearningMixin.save_checkpoint works.

Reply:
> Need to flip the inheritance I think? So that ActivelearningMixin.save_checkpoint works.

I think something on PL is not happy if it goes as second inheritance. I'll test it.